sdk/python/tests/integration/feature_repos/repo_configuration.py

```diff
@@ -509,6 +498,13 @@ def construct_test_environment(
             cache_ttl_seconds=1,
         )
 
+    if test_repo_config.online_store_creator:
```

Comment:
why did you move it down?

Reply:
it is not needed to move now, initially made some changes to incorporate that logic into the creator framework at that time there was some dependency but later that plan was dropped in my case.